pkg/http/api_test.go

```diff
@@ -5019,6 +5021,20 @@ func TestV1HealthzEndpoint(t *testing.T) {
 		assert.Equal(t, 204, resp.StatusCode)
 	})
 
+	t.Run("Healthz - 500 No AppId Match", func(t *testing.T) {
```

Comment:
Need empty ID case.

Reply:
This case has been available before

https://github.com/dapr/dapr/pull/6279/files#diff-3e12a060b83c3a6da274fd9de083fdec28ed255a49a5c347530eade7acee2e2bL5007-L5018